crates/ironrdp-web/Cargo.toml

```diff
@@ -40,6 +40,9 @@ tracing-web = "0.1"
 
 # Rendering
 softbuffer = { version = "0.3", default-features = false }
+png.workspace = true
+resize = "0.8"
```

Comment:
Question: looks like the `rayon` feature is enabled by default. Does it work in WASM? Should we disable the default features?

Reply:
You are right, It will be useless on web... I'll disable `rayon` feature